clang/test/dpct/cmake_migration/case_018/expected.txt

```diff
@@ -0,0 +1 @@
+dpct_compile(objs a.dp.cpp b.dp.cpp STATIC OPTIONS -O3 --profile)
```

Comment:
"STATIC OPTIONS -O3 --profile" is specific to cuda_compile, should be removed in SYCL side.

Reply:
Done! Added more test cases.